src/category_theory/quotient.lean

```diff
@@ -0,0 +1,101 @@
+import category_theory.functor
```

Comment:
Add a copyright header; just copy and paste from another file and update names and dates.